testsuite/python3.py

```diff
@@ -37,3 +37,9 @@ class Class:
 
     def m(self):
         xs: List[int] = []
+#: E227
+def bar(a, b)->int:
```

Comment:
It's saying that it's finding E225 instead of E227. That sems like a valid test failure at this point